src/main.zig

```diff
@@ -83,6 +83,10 @@ pub fn main() !void {
     var max_nano_seconds: u64 = std.time.ns_per_s * 5;
 
     var arg_i: usize = 1;
+    if (args.len < 2) {
+        try stdout.writeAll(usage_text);
+        std.process.exit(1);
+    }
```

Comment:
Better to put this after the arg parsing and check the number of commands. This code does not solve the problem for `poop -d 10`.

Reply:
right, i hadnt considered that case. fix pushed and found a bug where index OOB occurred if you did e.g. `poop -d` without any duration